solutions/task_4.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+"""
+TASK: Implement classes "Temperature", "Person" using Python descriptor protocols to execute unittests without errors.
+"""
+import re
+import unittest
+from datetime import datetime
+
+
+class Temperature(object):
+
+    def __init__(self, value=0):
+        self.fahrenheit = value
+
+    def get_celsius(self):
+        return (self.fahrenheit - 32)/1.8
+
+    def set_celsius(self, value):
+        self.fahrenheit = value*1.8 + 32
+
+    celsius = property(get_celsius, set_celsius)
```

Comment:
Why do you use ```property``` here instead of descriptor protocol? Just a question. I think you can leave it to have such example.

Reply:
According to the information I've read about descriptor protocol property common usage is to provide a simple way to change some attribute behaviour. So I've decided that task with temperatures is not so complicate and property is a good way to receive elegant solution fast. Also I would like to have an example of property implementation and this is another reason why this function has been used.